<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>teacherform</class>
 <widget class="QWidget" name="teacherform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>试题管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>类型：</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="cbxType">
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C++</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Java</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C#</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>题目列表：</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="4">
          <widget class="QListView" name="lvTitle"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>测试题：</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="4">
          <widget class="QListView" name="lvContent"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>答案：</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="cbxAns">
           <item>
            <property name="text">
             <string>A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>B</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>C</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>D</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="btnAdd">
           <property name="text">
            <string>添加</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="btnDel">
           <property name="text">
            <string>删除</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="btnRefresh">
           <property name="text">
            <string>刷新</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="btnQuit">
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>118</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>成绩管理</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0" colspan="4">
        <widget class="QTableView" name="tvScore_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>学生学号：</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="leStdNum_2"/>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>79</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnSearch_2">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>密码修改</string>
      </attribute>
      <widget class="QLineEdit" name="leOldPwd_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>50</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNewPwd_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>110</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNewPwd2_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>170</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="btnOk_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>250</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnQuit_3">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>250</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>旧密码：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>新密码</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
